handbook/company/software-engineer-windows.md

```diff
@@ -0,0 +1,66 @@
+# 🚀 Software Engineer (Windows/Go)
+
+## Let's start with why we exist. 📡
+
+Ever wondered if your employer is monitoring your work computer?
+
+Organizations make huge investments every year to keep their laptops and servers online, secure, compliant, and usable from anywhere. This is called "device management".
+
+At Fleet, we think it's time device management became [transparent](https://fleetdm.com/transparency) and [open source](https://fleetdm.com/handbook/company#open-source).
+
+
+## About the company 🌈
+
+You can read more about the company in our [handbook](https://fleetdm.com/handbook/company), which is public and open to the world.
+
+Fleet Device Management Inc. is a [recently-funded](https://techcrunch.com/2022/04/28/fleet-nabs-20m-to-enable-enterprises-to-manage-their-devices/) Series A startup founded and backed by the same people who created osquery, the leading open source security agent. Today, osquery is installed on millions of laptops and servers, and it is especially popular with [enterprise IT and security teams](https://www.linuxfoundation.org/press/press-release/the-linux-foundation-announces-intent-to-form-new-foundation-to-support-osquery-community).
+
+
+## Your primary responsibilities 🔭
+
+At Fleet, you will get the chance to…
+
+- 🧑‍🔬 Design, develop, test, and maintain a state-of-the-art Golang application that includes robust APIs to support mobile and desktop clients.
```

Comment:
How about making this a bit more focused on the opportunity to lead Windows MDM development? Maybe something like: 

- 🧑‍🔬 Lead the architecture, development, and implementation of Fleet's Windows MDM offering.

Reply:
Love it, changed.